### PR TITLE
Publish page component

### PR DESCRIPTION
**Pull request by NaomiLea**

### CODE REVIEW

File: frontend/storybook/src/components/entry/publish/pages/publish-two.ts [hunk 1]
```diff
@@ -1,86 +1,71 @@
...
-//       const STR_EMBED ="Embed this JIG";
-//       const STR_EMBEDICON ="Icn_Embed.svg";
-
-      
+    const DEFAULT_ARGS:PublishArgs = {
+       closed:false,
+       embedclosed:false,
+      }
 
+      const STR_SHARE = "share the JIG";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/publish/pages/publish-two.ts:28]: Shouldn't these be in an element?

File: frontend/storybook/src/components/entry/publish/pages/publish-one.ts [hunk 2]
```diff
@@ -34,23 +31,19 @@ export default {
...
       }
 
-const STR_TITLE ="Settings and JIG info.";
-const STR_SUBTITLE = "Last step before publishing";
+
 const STR_BTNLABEL = "Publish JIG";
 const STR_IMGTHUMBNAIL = "red-sea-book.png";
 const STR_SLIDERLABEL = "My JIG is public";
 const STR_NAME = "JIG’s name";
 const STR_LANGUAGE = "Language of instructions";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/publish/pages/publish-one.ts:39]: shouldn't some of these be in an element (like AGE, LANGUAGE)?

**Review by dakom — CHANGES_REQUESTED** [review]

File: frontend/storybook/src/components/entry/publish/pages/publish-two.ts (hunk 1 above)
Comment (NaomiLea) [reply frontend/storybook/src/components/entry/publish/pages/publish-two.ts:28]: Its the string for a rectangle button so I believe its best to be kept here

File: frontend/storybook/src/components/entry/publish/pages/publish-one.ts (hunk 2 above)
Comment (NaomiLea) [reply frontend/storybook/src/components/entry/publish/pages/publish-one.ts:39]: these are used for dropdowns that are elements already

**Pull request merged**